git/week2/main.js

```diff
@@ -0,0 +1,49 @@
+
+//console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+//console.log(productsUl);
+
+let e=document.querySelector(".country>select")
+
+e.addEventListener("change", x=>{
+        let targetCountry= x.target.value;
+        let myCoutry=products.filter(x=>
+            {
+            
+          if(x.shipsTo==targetCountry)
```

Comment:
shipsTo is a string like `Denmark` but the target country value is `<option value="denmark">Denmark</option>` or `denmark` in this case.